File: .ProgramingBasics/ComplexConditionalStatements/10.AnimalType/src/AnimalType.java
import java.util.Scanner;

public class AnimalType {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();

        switch (input) {
            case "dog":
                System.out.println("mammal");
                break;
            case "crocodile":
            case "tortoise":
            case "snake":
                System.out.println("reptile");
                break;
            default:
                System.out.println("unknown");
        }
    }
}

File: ProgramingBasics/ComplexConditionalStatements/10.AnimalType/10.AnimalType/AnimalType.cs
using System;

namespace _10.AnimalType
{
    public class AnimalType
    {
        public static void Main(string[] args)
        {
            var input = Console.ReadLine();
            var result = string.Empty;

            switch (input)
            {
                case "dog":
                    result = "mammal";
                    break;
                case "crocodile":             
                case "tortoise":
                case "snake":
                    result = "reptile";
                    break;
                default:
                    result = "unknown";
                    break;
            }
        }
    }
}
